pkg/services/postgresql/catalog.go

```diff
@@ -105,33 +102,31 @@ func (m *module) GetCatalog() (service.Catalog, error) {
 				},
 			}),
 		),
-		//db only
+		// database only
```

Comment:
Pervasive: I'm opting to use "database" in lieu of "db" wherever I can because this is how I want users or contributors to talk-- because "db" is too easily confused with "dbms" when written or read, spoken or heard.